neqo-transport/src/connection.rs

```diff
@@ -2084,6 +2093,7 @@ impl Connection {
     /// to retransmit the frame as needed.
     fn handle_lost_packets(&mut self, lost_packets: &[SentPacket]) {
         for lost in lost_packets {
+            qlog::packet_lost(&mut self.qlog.borrow_mut(), &lost);
```

Comment:
Unfortunately, this is going to be wrong.  We use this function for marking data as lost for reasons other than lost packets.  There are two places that we genuinely mark packets as lost: when we handle an ACK and when we run the time-based loss recovery.  This is also called for discarded 0-RTT, PTO, and Retry.

However, I also think that qlog is completely wrong in what it is asking for here.  It should be recording just the packet number that was marked lost and relying on the logging for that packet to determine what data was lost along with that.

Reply:
ok, will do.